src/geocel/vg/detail/BVHNavigator.hh

```diff
@@ -33,30 +33,29 @@ namespace detail
 class BVHNavigator
 {
   public:
-    using Precision = vecgeom::Precision;
-    using Vector3D = vecgeom::Vector3D<vecgeom::Precision>;
-    using VPlacedVolumePtr_t = vecgeom::VPlacedVolume const*;
+    using VgPlacedVol = VgPlacedVolume<MemSpace::native>;
+    using Nav = VgNavState;
 
-    static constexpr Precision kBoundaryPush = 10 * vecgeom::kTolerance;
+    static constexpr vg_real_type kBoundaryPush = 10 * vecgeom::kTolerance;
 
-    CELER_FUNCTION static VPlacedVolumePtr_t
-    LocatePointIn(vecgeom::VPlacedVolume const* vol,
-                  Vector3D const& point,
-                  vecgeom::NavigationState& path,
+    CELER_FUNCTION static void
+    LocatePointIn(VgPlacedVol const* vol,
+                  VgReal3 const& point,
+                  Nav& path,
```

Comment:
Document path as output / side effect?

Reply:
These have always been side effects, and I've updated the name to NavState.